src/test.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const value: string;
+  export default value;
+}
```

Comment:
I think you can do that, as Serhii said in the Slack channel. Use components for icons.